manywheel/build.sh

```diff
@@ -46,29 +46,25 @@ else
     echo "CUDA $CUDA_VERSION Detected"
 fi
 
-export TORCH_CUDA_ARCH_LIST="3.7+PTX;5.0"
-if [[ $CUDA_VERSION == "9.0" ]]; then
-    export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;7.0"
-elif [[ $CUDA_VERSION == "9.2" ]]; then
-    export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0"
-    # ATen tests can't build with CUDA 9.2 and the old compiler used here
-    EXTRA_CAFFE2_CMAKE_FLAGS+=("-DATEN_NO_TEST=ON")
-elif [[ $CUDA_VERSION == "10.0" ]]; then
-    export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0;7.5"
-    # ATen tests can't build with CUDA 10.0 maybe???? (todo) and the old compiler used here
-    EXTRA_CAFFE2_CMAKE_FLAGS+=("-DATEN_NO_TEST=ON")
-elif [[ $CUDA_VERSION == "10.1" ]]; then
-    export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0;7.5"
-    # ATen tests can't build with CUDA 10.1 maybe???? (todo) and the old compiler used here
-    EXTRA_CAFFE2_CMAKE_FLAGS+=("-DATEN_NO_TEST=ON")
-elif [[ $CUDA_VERSION == "10.2" ]]; then
-    export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0;7.5"
-    # ATen tests can't build with CUDA 10.1 maybe???? (todo) and the old compiler used here
-    EXTRA_CAFFE2_CMAKE_FLAGS+=("-DATEN_NO_TEST=ON")
-else
-    echo "unknown cuda version $CUDA_VERSION"
-    exit 1
-fi
+export TORCH_CUDA_ARCH_LIST="5.0"
```

Comment:
I'd rather keep 3.7+PTX than 5.0.
3.7 is used much more than 5.2 / 5.x

Reply:
I'll try with that then

Reply:
# Removing just `5.0`

So I just ran the experiment for removing just `5.0` and then adding the `BUILD_CAFFE2_OPS=OFF` and we are over the size limit.

![Screen Shot 2020-07-08 at 11 50 29 AM](https://user-images.githubusercontent.com/1700823/86958690-69b07c00-c111-11ea-93b6-3f5cd7f13488.png)

From: https://app.circleci.com/pipelines/github/pytorch/pytorch/188853/workflows/44ec3b8b-7f9a-48a3-b1b5-2f6649cd70e1/jobs/6163364/steps

# Removing just `3.7+PTX`

Conversely when removing just `3.7+PTX` we were under the size limit:
![Screen Shot 2020-07-07 at 3 36 52 PM](https://user-images.githubusercontent.com/1700823/86958775-86e54a80-c111-11ea-9cf5-e66a004ef3d6.png)

From: https://app.circleci.com/pipelines/github/pytorch/pytorch/188605/workflows/489c448e-f5e2-4e2d-a789-50b848431b82/jobs/6150612

# Removing both and 6.1

When removing both and 6.1 we gain a bunch of leeway

![image](https://user-images.githubusercontent.com/1700823/86958879-b7c57f80-c111-11ea-8690-f425aac7fe8d.png)

From: https://app.circleci.com/pipelines/github/pytorch/pytorch/188553/workflows/8cc2a6e4-6ea9-4847-a4a7-b638d9c53bc9/jobs/6147507/steps

Reply:
So from what I'm seeing is that we have to remove _at least_ `3.7+PTX` from CUDA 10.2 since that's what actually gets us under the limit.

Reply:
Can I ask you to run one more experiment: how much would we win by replacing `3.7+PTX` with just `3.7`?

Reply:
Looks like removing the PTX also helps but still puts us over the limit

<img width="898" alt="Screen Shot 2020-07-08 at 3 33 45 PM" src="https://user-images.githubusercontent.com/1700823/86977923-89a46780-c132-11ea-9814-25b836eadfdd.png">

From: https://app.circleci.com/pipelines/github/pytorch/pytorch/188939/workflows/fb6b45cd-eb3a-4f34-8b47-98c9a359b7f9/jobs/6167279/steps